econml/tests/test_cate_interpreter.py

```diff
@@ -49,7 +50,7 @@ def test_cate_uncertainty_needs_inference(self):
         X = np.random.normal(size=(n, 4))
         T = np.random.binomial(1, 0.5, size=(n,))
         Y = (2 * (X[:, 0] > 0) - 1) * T.flatten()
-        est = LinearDML(discrete_treatment=True)
+        est = LinearDML(model_y=LinearRegression(), model_t=LogisticRegression(), discrete_treatment=True)
```

Comment:
Has something changed in the implementation that necessitates these changes to the test?

Reply:
We're running this too many times and it's better to use un-penalized to make it run in reasonable time. That's not what we're testing here, was my reasoning.